api/src/main/java/com/velocitypowered/api/plugin/PluginDescription.java

```diff
@@ -0,0 +1,75 @@
+package com.velocitypowered.api.plugin;
+
+import com.google.common.collect.ImmutableSet;
+import com.velocitypowered.api.plugin.meta.PluginDependency;
+
+import java.nio.file.Path;
+import java.util.Collection;
+import java.util.Optional;
+import java.util.Set;
+import java.util.regex.Pattern;
+
+/**
+ * Represents metadata for a specific version of a plugin.
+ */
+public interface PluginDescription {
```

Comment:
`PluginDescriptor` would be a better name for this.

Reply:
This does sound more like an internal implementation detail.